### PR TITLE
Fix the topcrash test and mockup requests to socorro

### PR DESCRIPTION
**Pull request by suhaibmujahid**
This should fix the tests for #1659

## Checklist

- [ ] Type annotations added to new functions
- [ ] Docs added to functions touched in main classes
- [x] Dry-run produced the expected results
- [ ] The [`to-be-announced`](https://github.com/mozilla/relman-auto-nag/labels/to-be-announced) tag added if this is worth announcing

### CODE REVIEW

File: auto_nag/tests/test_topcrash.py [hunk 1]
```diff
@@ -16,4 +22,4 @@ def test_get_blocked_signatures(self):
 
         assert "OOM | small" in signatures
         assert "IPCError-browser | ShutDownKill" in signatures
-        assert "EMPTY: no crashing thread identified; EmptyMinidump" in signatures
+        assert "EMPTY: no frame data available; StreamSizeMismatch" in signatures
```
Comment (calixteman) [inline auto_nag/tests/test_topcrash.py:25]: Why did you change the signature here ?

Comment (suhaibmujahid) [reply auto_nag/tests/test_topcrash.py:25]: It changed from the source, thus I mock the request to avoid the need to update this again in the future.

**Review by calixteman — APPROVED** [review]
